make/mapfiles/libjava/reorder-x86

```diff
@@ -10,6 +10,8 @@ text: .text%collapse: OUTPUTDIR/canonicalize_md.o;
 text: .text%Java_java_lang_Object_registerNatives;
 text: .text%Java_java_lang_System_registerNatives;
 text: .text%Java_java_lang_Thread_registerNatives;
+text: .text%Java_com_alibaba_tenant_TenantGlobals_getTenantFlags;
+text: .text%Java_com_alibaba_tenant_TenantGlobals_getTenantFlags;
```

Comment:
the two added lines seem the same, please delete one.

Reply:
done